tests/integration/test_utils/data_source_config/base.py

```diff
@@ -48,6 +49,7 @@ def create_batch_setup(
         request: FixtureRequest,
         data: pd.DataFrame,
         extra_data: Mapping[str, pd.DataFrame],
+        context: Union[AbstractDataContext, None] = None,
```

Comment:
Why make this nullable? Seems like it's just opening us up to mistakes down the road.

Reply:
sure, just took that approach because it was fastest, but i think you're right. updated in [2b5e77d](https://github.com/great-expectations/great_expectations/pull/11138/commits/2b5e77d684d5fed0ecf792ceab259c0562b38f3a)